sync-deps/src/main/java/com/spotify/syncdeps/model/MavenCoords.java

```diff
@@ -66,21 +70,38 @@ public static String groupRelativePackageName(final String groupId) {
         .collect(joining("/"));
   }
 
+  public static MavenCoords create(final String groupId, final String artifactId) {
+    return create(groupId, artifactId, Optional.empty());
+  }
+
   @JsonCreator
   public static MavenCoords create(
       @JsonProperty("groupId") final String groupId,
-      @JsonProperty("artifactId") final String artifactId) {
-    return new AutoValue_MavenCoords(groupId, artifactId);
+      @JsonProperty("artifactId") final String artifactId,
+      @JsonProperty("classifier") final Optional<String> classifier) {
```

Comment:
I would like to assume that Jackson does not know about `Optional<...>`, so make this `@Nullable` and use `Optional.fromNullable()`